src/Simulation/EntryPointDriver/IEntryPoint.cs

```diff
@@ -16,10 +17,13 @@ namespace Microsoft.Quantum.EntryPointDriver
     /// Contains entry point properties needed by the command-line interface and allows the entry point to use
     /// command-line arguments. The implementation of this interface is code-generated.
     /// </remarks>
-    /// <typeparam name="TIn">The entry point's argument type.</typeparam>
-    /// <typeparam name="TOut">The entry point's return type.</typeparam>
-    public interface IEntryPoint<TIn, TOut>
+    public interface IEntryPoint
     {
+        /// <summary>
+        /// The name of the entry point operation.
```

Comment:
nit: The entry point can technically be a function as well.

```suggestion
        /// The name of the entry point.
```

Reply:
done